Makefile

```diff
@@ -0,0 +1,113 @@
+BACKEND?=docker
```

Comment:
After yesterday conversation, maybe better to move this to a single script, or either reuse the `.envrc` from `cos-toolkit`? wdyt?

Reply:
Probably we can try to simplify the Makefile parameters slightly, even it is already quite simple and additionally document how to run local build without the makefile using a containerized luet, a three steps commands (build, create repo and geniso).